src/request/components/request-view.scss

```diff
@@ -377,54 +377,96 @@
 }
 
 .tab-logs {
-    .fa-svg {
-        height: 12px;
-        width: 12px;
-        margin-right: 5px;
-        vertical-align: middle;
+    .tab-logs-message-count {
+        font-family: 'Segoe UI', 'Selawik Semibold', Tahoma, Geneva, Verdana, sans-serif;
+        font-size: 175%;
     }
-}
 
-.tab-logs-data-replaced-region {
-    font-weight: bold;
-}
+    .tab-logs-table {
+        table-layout: fixed;
+        
+        .fa-svg {
+            height: 12px;
+            width: 12px;
+            margin-right: 3px;
+            vertical-align: middle;
+        }
 
-.tab-logs-message-count {
-    font-family: 'Segoe UI', 'Selawik Semibold', Tahoma, Geneva, Verdana, sans-serif;
-    font-size: 175%;
-}
+        .tab-logs-table-message {
+            display: flex;
+            flex-direction: row;
+            
+            .tab-logs-table-message-icon {
+                visibility: hidden;               
+            }
 
-.tab-logs-data-default {
-    font-weight: normal;
-    color: #333333; // grey
-    vertical-align: middle;
-}
+            .tab-logs-table-message-icon-expandable {
+                @extend .tab-logs-table-message-icon;
+                
+                visibility: visible;               
+            }
 
-.tab-logs-data-warning {
-    @extend .tab-logs-data-default;
-    
-    color: #D35400; // orange
-    
-    .fa-svg {
-        .fa-svg-path {        
-          fill: #D35400; // orange
+            &:hover {
+                .tab-logs-table-message-icon {
+                    visibility: visible;               
+                }
+            }
+ 
+            .tab-logs-table-message-object {
+                overflow: auto;
+                
+                > pre {
+                        margin: 0;
+                        
+                        > code {
+                            background: transparent;
+                            padding: 0;
+                        }
+                    }
+            }
```

Comment:
Just be careful about how nested these get. Its easy to forget that these get blown up to rather long selectors when they are compiled. Additionally kinda against best practice to go more than one/two levels deep in cases like this. The fact that they are prefixed with `tab-logs` should be enough specificity, otherwise you end up getting into specificity wars as the app gets bigger.